main.nf

```diff
@@ -990,8 +990,8 @@ workflow.onError {
         }
 
         focal_species_line = file("${configfile}").readLines().find{ it =~ /focal_species/ }
-        if ( focal_species_line.split().size() == 3 ) {
-            species_name = focal_species_line.split()[2].strip()
+        if ( focal_species_line.split("=").size() == 2 ) {
```

Comment:
Hmm, is `size() == 2` the correct case here? Wouldn't `focal_species = ` also result in `size() == 2`?

Reply:
Since it splits by "=", the "=" itself doesn't appear anymore in the result (which is [focal_species, elaeis]), so the list size is 2.

Reply:
Yes, I understand that '=' is not included. But I was referring to the case `focal_species = ` (with a space after the '=', like it is generated by default from our code when the config file does not exist yet), then you also get size 2, "focal_species" and " ".

Reply:
Maybe add a `strip()` or `trim()`.

Reply:
Oh, didn't see that! Added the strip():
`if ( focal_species_line.strip().split("=").size() == 2 )`